docs/pydoc/renderers.py

```diff
@@ -133,3 +133,16 @@ def _frontmatter(self) -> str:
             slug=self.slug,
             order=self.order,
         )
+
+
+@dataclasses.dataclass
+class ReadmePreviewRenderer(ReadmeRenderer):
+    """
+    This custom Renderer behaves just like the ReadmeRenderer but renders docs with the hardcoded version 2.0 to generate correct category ids.
+    """
+
+    def _doc_version(self) -> str:
+        """
+        Returns the hardcoded docs version 2.0.
+        """
+        return "v2.0"
```

Comment:
Should this perhaps be 2.x ? We'll have to change it all the time, no?

Reply:
Maybe we can grab version from somewhere?

Reply:
At that point I would expect to not have the preview folder anymore. And we would also have API docs for the stable release for 2.x plus unstable main branch. That's why I interpreted the issue to be about 2.0 only.